apps/src/componentLibrary/dropdown/checkboxDropdown/CheckboxDropdown.tsx

```diff
@@ -54,6 +55,14 @@ export interface CheckboxDropdownProps extends AriaAttributes {
       | React.MouseEvent<HTMLButtonElement>
       | React.MouseEvent<HTMLAnchorElement>
   ) => void;
+  /** CheckboxDropdown helper message */
+  helperMessage?: string;
+  /** CheckboxDropdown helper icon */
+  helperIcon?: FontAwesomeV6IconProps;
+  /** CheckboxDropdown error message */
+  errorMessage?: string;
+  /** Style the dropdown as a form field */
+  styleAsFormField?: boolean;
```

Comment:
Worth creating some base interface to share these options?

Reply:
Great idea, thanks a lot for the hint!
Just checked, storybook still fetches the documentation for these props this way which is awesome!